src/server.ts

```diff
@@ -1,11 +1,12 @@
 import path from 'path';
-
+var bodyParser = require('body-parser');
```

Comment:
❗
Я понимаю, что это был быстрый копипаст, но не `var` же :D
```suggestion
import bodyParser from 'body-parser';
```